packages/xdl/src/Exp.ts

```diff
@@ -126,9 +127,25 @@ export async function extractTemplateAppAsync(
   targetPath: string,
   config: AppJSONConfig | BareAppConfig
 ) {
-  let tarStream = await pacote.tarball.stream(templateSpec, {
-    cache: path.join(UserSettings.dotExpoHomeDirectory(), 'template-cache'),
-  });
+  await pacote.tarball.stream(
+    templateSpec,
+    () => {
+      return extractTemplateAppAsyncImpl(templateSpec, targetPath, config);
```

Comment:
The 4th argument (`tarStream`) seems to be missing here?

```
@expo/xdl: src/Exp.ts(133,14): error TS2554: Expected 4 arguments, but got 3.
```

Reply:
Woops indeed! Are there tests I can easily run to double-check that everything looks fine?

Reply:
The tests for `expo-cli` (they can be run with `yarn test` in `packages/expo-cli`) test the `expo init` command, and they all passed in CircleCI, so it looks fine now. 👍